py/torch_migraphx/fx/tracer/acc_tracer/acc_ops.py

```diff
@@ -33,6 +33,7 @@
 
 import torch
 from typing import cast, Iterable, List, Sequence
+import torchvision
```

Comment:
```suggestion
try:
    import torchvision
except ImportError:
    pass
```
This is not a mandatory prerequisite for torch_migraphx so wrap it in a try/except